test/testprocess.c

```diff
@@ -0,0 +1,159 @@
+#include <SDL3/SDL.h>
+#include <SDL3/SDL_test.h>
```

Comment:
This needs `SDL_main.h` for psp, I think.

```suggestion
#include <SDL3/SDL.h>
#include <SDL3/SDL_main.h>
#include <SDL3/SDL_test.h>
```